dashboard/app/views/levels/_admin.html.haml

```diff
@@ -35,6 +32,10 @@
                   = text_field_tag :name, @level.name
                   = hidden_field_tag :authenticity_token, form_authenticity_token
                   = submit_tag 'Clone'
+              - content_for(:head) do
+                %script{src: minifiable_asset_path('js/levelbuilder.js')}
+              :javascript
+                window.levelbuilder.ajaxSubmit('.clone_level');
```

Comment:
In this case it's binding this specific element so I'm tempted not to clean this up right now.